project.clj

```diff
@@ -24,7 +24,9 @@
                  [nubank/state-flow "5.14.4"]
                  [clj-http "3.12.3"]
                  [clj-http-fake "1.0.3"]
+                 [hashp "0.2.2"]
                  [io.replikativ/datahike "0.6.1531" :exclusions [com.cognitect/transit-clj com.cognitect/transit-java org.clojure/clojurescript com.google.guava/guava com.taoensso/encore fress org.lz4/lz4-java com.taoensso/timbre]]]
+  :injections [(require 'hashp.core)]
```

Comment:
Added the `hashp` lib, now you can just use `#p` to debug the output of any function in any point of the code

https://github.com/weavejester/hashp